Cargo.toml

```diff
@@ -42,8 +42,9 @@ serde_json = "1.0.85"
 serde_pack = { version = "1.1.0", package = "rmp-serde" }
 surrealdb = { path = "lib" }
 thiserror = "1.0.35"
-tokio = { version = "1.21.1", features = ["macros", "signal"] }
+tokio = { version = "1.21.1", features = ["macros", "signal", "io-std"] }
```

Comment:
What is the `io-std` feature for @mathe42 ?

Reply:
This allowes https://github.com/surrealdb/surrealdb/blob/443b81b66ac5aefc6117f87b85ef64325440ca92/src/lsp/mod.rs#L30-L31 there might be ways to avoid to include that. Not sure.

Reply:
Is this the recommended way of communicating with the LSP?

Reply:
Yes. Deno uses the exact setup. With VSCode we can directly integrate it (and have multiple instances if needed).

Reply:
see https://github.com/mathe42/surrealql_vscode/blob/master/src/extension.ts for implementation.